<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>addBookDialog</class>
    <widget class="QDialog" m_name="addBookDialog">
        <property m_name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>658</width>
                <height>354</height>
            </rect>
        </property>
        <property m_name="windowTitle">
            <string>Add Book</string>
        </property>
        <property m_name="windowIcon">
            <iconset resource="../resources/images.qrc">
                <normaloff>:/icons/books_icon.png</normaloff>:/icons/books_icon.png
            </iconset>
        </property>
        <layout class="QVBoxLayout" m_name="verticalLayout">
            <item>
                <widget class="QLabel" m_name="labelTitle">
                    <property m_name="text">
                        <string>Add Book</string>
                    </property>
                    <property m_name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QFrame" m_name="frame">
                    <property m_name="styleSheet">
                        <string notr="true">QFrame
                            {
                            border: 0px;
                            }
                        </string>
                    </property>
                    <property m_name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property m_name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QGridLayout" m_name="gridLayout">
                        <item row="6" column="1">
                            <widget class="QLineEdit" m_name="textSize">
                                <property m_name="enabled">
                                    <bool>false</bool>
                                </property>
                            </widget>
                        </item>
                        <item row="8" column="1">
                            <widget class="QLineEdit" m_name="textTags"/>
                        </item>
                        <item row="3" column="1">
                            <widget class="QLineEdit" m_name="textExtension">
                                <property m_name="enabled">
                                    <bool>false</bool>
                                </property>
                                <property m_name="cursor">
                                    <cursorShape>PointingHandCursor</cursorShape>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLineEdit" m_name="textFolder"/>
                        </item>
                        <item row="5" column="1">
                            <widget class="QLineEdit" m_name="textGenre"/>
                        </item>
                        <item row="4" column="0">
                            <widget class="QLabel" m_name="labelAuthor">
                                <property m_name="text">
                                    <string>Author</string>
                                </property>
                            </widget>
                        </item>
                        <item row="7" column="0">
                            <widget class="QLabel" m_name="labelPages">
                                <property m_name="text">
                                    <string>Pages</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QLineEdit" m_name="textPath"/>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" m_name="textName"/>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLabel" m_name="labelPath">
                                <property m_name="text">
                                    <string>Path</string>
                                </property>
                            </widget>
                        </item>
                        <item row="6" column="0">
                            <widget class="QLabel" m_name="labelSize">
                                <property m_name="text">
                                    <string>Size</string>
                                </property>
                            </widget>
                        </item>
                        <item row="8" column="0">
                            <widget class="QLabel" m_name="labelGroups">
                                <property m_name="text">
                                    <string>Tags</string>
                                </property>
                            </widget>
                        </item>
                        <item row="5" column="0">
                            <widget class="QLabel" m_name="labelGenre">
                                <property m_name="text">
                                    <string>Genre</string>
                                </property>
                            </widget>
                        </item>
                        <item row="7" column="1">
                            <widget class="QLineEdit" m_name="textPages"/>
                        </item>
                        <item row="2" column="2">
                            <widget class="QPushButton" m_name="buttonBrowse">
                                <property m_name="minimumSize">
                                    <size>
                                        <width>40</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property m_name="maximumSize">
                                    <size>
                                        <width>40</width>
                                        <height>16777215</height>
                                    </size>
                                </property>
                                <property m_name="cursor">
                                    <cursorShape>PointingHandCursor</cursorShape>
                                </property>
                                <property m_name="text">
                                    <string>...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" m_name="labelFolder">
                                <property m_name="text">
                                    <string>Folder</string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="1">
                            <widget class="QLineEdit" m_name="textAuthor"/>
                        </item>
                        <item row="0" column="0">
                            <widget class="QLabel" m_name="labelName">
                                <property m_name="text">
                                    <string>Name</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0">
                            <widget class="QLabel" m_name="labelExtension">
                                <property m_name="text">
                                    <string>Extension</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" m_name="horLayButtons">
                    <property m_name="spacing">
                        <number>6</number>
                    </property>
                    <property m_name="leftMargin">
                        <number>0</number>
                    </property>
                    <property m_name="rightMargin">
                        <number>0</number>
                    </property>
                    <item>
                        <spacer m_name="horizontalSpacer">
                            <property m_name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property m_name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" m_name="buttonAdd">
                            <property m_name="minimumSize">
                                <size>
                                    <width>80</width>
                                    <height>0</height>
                                </size>
                            </property>
                            <property m_name="maximumSize">
                                <size>
                                    <width>80</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property m_name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property m_name="text">
                                <string>Add</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" m_name="buttonClear">
                            <property m_name="minimumSize">
                                <size>
                                    <width>80</width>
                                    <height>0</height>
                                </size>
                            </property>
                            <property m_name="maximumSize">
                                <size>
                                    <width>80</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property m_name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property m_name="text">
                                <string>Clear</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" m_name="buttonCancel">
                            <property m_name="minimumSize">
                                <size>
                                    <width>80</width>
                                    <height>0</height>
                                </size>
                            </property>
                            <property m_name="maximumSize">
                                <size>
                                    <width>80</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property m_name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property m_name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer m_name="horizontalSpacer_2">
                            <property m_name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property m_name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <tabstops>
        <tabstop>textName</tabstop>
        <tabstop>textFolder</tabstop>
        <tabstop>textPath</tabstop>
        <tabstop>buttonBrowse</tabstop>
        <tabstop>textExtension</tabstop>
        <tabstop>textAuthor</tabstop>
        <tabstop>textGenre</tabstop>
        <tabstop>textSize</tabstop>
        <tabstop>textPages</tabstop>
        <tabstop>textTags</tabstop>
        <tabstop>buttonAdd</tabstop>
        <tabstop>buttonClear</tabstop>
        <tabstop>buttonCancel</tabstop>
    </tabstops>
    <resources>
        <include location="../resources/images.qrc"/>
    </resources>
    <connections/>
</ui>
